SickSangHae/ViewModel/UpdateItemViewModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  UpdateItemViewModel.swift
+//  SickSangHae
+//
+//  Created by 최효원 on 2023/07/24.
+//
+
+import SwiftUI
+
+class UpdateItemViewModel: ObservableObject {
```

Comment:
이후에 더이상 상속되지 않는다면 final을 붙이면 상속되지않도록 보장할 수 있어요 ! 👍. 
```suggestion
final class UpdateItemViewModel: ObservableObject {
```